### ISSUE TITLE
 Yandex Images - Search by image.

### ISSUE THREAD

**Issue by UjwalKandi** [UjwalKandi/Issue-Bookmarks#350]
**TL;DR** - Identify objects around you, Learn more about art, languages, and technology. Search for something similar or unique.

### Image search
Yandex can search for images based on an image you upload. This can be an entire image or a fragment of an image. The image can either be from the internet or stored on your computer or other devices.

Image search is based on computer vision algorithms. The search results may contain exact copies of the source image and/or images that differ in minor ways from the original. For example, it may be different photographs of the same monument or building.

**Website**:
https://yandex.com/images/
https://yandex.com/images/?rpt=imageview
